programmer/tinyprog/__main__.py

```diff
@@ -331,7 +331,7 @@ def progress(info):
                 if not fpga.program_bitstream(addr, bitstream):
                     sys.exit(1)
 
-            fpga.boot()
+            # fpga.boot()
```

Comment:
Why is this commented out?  The programmer is supposed to reboot the target FPGA board into the user design after successfully programming the user bitstream.

Reply:
It's just commented out because I don't want to leave bootloader after each
upload, e.g. for initialization I need to upload 3-4 times to different flash address
primariy image, golden image, jump commands, metadata....

If boot is executed after first upload, then this procedure can't be
conveniently automated in a script and user will get confused how to
initialize.

Reply:
Ok, so the solution here is to add a command-line flag to disable automatic reboot after programming a user bitstream.  I would call it `--no-boot`.

Reply:
yes, that's the good solution